kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/NetworkTabsUi.java

```diff
@@ -328,6 +346,7 @@ private void showWirelessTabs() {
         this.wirelessTabAnchorItem.setEnabled(true);
 
         insertTab(this.wirelessTabAnchorItem);
+        insertTab(this.net8021xTabAnchorItem);
```

Comment:
We should show the tab and insert the anchor item only if we are on generic profiles that have `NetworkManager`, i.e. only if `this.isNet2` is true. Look at the `updateTabs()` method.